tracing-flame/src/lib.rs

```diff
@@ -235,6 +246,16 @@ where
             out: self.out.clone(),
         }
     }
+
+    pub fn filter_empty(mut self) -> Self {
```

Comment:
In most of the other `tracing` crates, this kind of builder takes a `bool` so that the feature can be turned off _or_ on --- this way, the user can explicitly _disable_ the setting as well as enable it, so they don't have to rely on it being off by default:
https://github.com/tokio-rs/tracing/blob/1c2dffd0582b27c297a5a6292da3d871be4f7b11/tracing-subscriber/src/fmt/mod.rs#L490-L510

Might be good to do that here for consistency?